external-crates/move/crates/move-compiler/tests/move_check/parser/function_without_body.exp

```diff
@@ -1,9 +1,9 @@
 error[E01002]: unexpected token
-  ┌─ tests/move_check/parser/function_without_body.move:3:13
+  ┌─ tests/move_check/parser/function_without_body.move:5:1
   │
-3 │     fun fn();
-  │             ^
-  │             │
-  │             Unexpected ';'
-  │             Expected '{'
+5 │ 
+  │ ^
+  │ 
+  │ Unexpected end-of-file
+  │ Expected a module member: 'spec', 'use', 'friend', 'const', 'fun', or 'struct'
```

Comment:
This seems like a bad regression. Probably something to do with your attempt to only spit out one error per loc?

Reply:
It was an easy fix and fortunately had nothing to do with one error per loc part.